### DISCUSSION TITLE
Install manpage when building from source

### DISCUSSION THREAD

**Discussion by yetyetanotherusername** [ghostty-org/ghostty#2724] [Feature Requests, Ideas] [answered]
Building from source as documented in the readme currently leads to an installation without the manpage. Would it be ok to add `-Demit-docs` argument so that new users don't end up with no manpage?

If yes, perhaps also pandoc should be added to the build requirements, as otherwise the build fails.

**Comment by mitchellh** [comment]
`emit-docs` is already true if `pandoc` is available. We should probably document this for the packaging directions though.

**Reply by mitchellh** [reply, marked as answer]
#2726
